Content/1.1.3 - PreAuthorization Review/preAuthorizationReview.json

```diff
@@ -16,5 +16,13 @@
 	"preAuthorizationReview_timeFormat_seconds": "%d seconds",
 	"preAuthorizationReview_slideToSign": "Slide to Sign and Return",
 	"preAuthorizationReview_rawManifestAlert_title": "Warning",
-	"preAuthorizationReview_rawManifestAlert_message": "This is a complex pre-authorization that cannot be summarized - the raw pre-authorization manifest will be shown. Do not sign and return unless you understand the contents."
+	"preAuthorizationReview_rawManifestAlert_message": "This is a complex pre-authorization that cannot be summarized - the raw pre-authorization manifest will be shown. Do not sign and return unless you understand the contents.",
+	"preAuthorizationReview_unknownStatus_title": "Pre-Authorization Sent",
+	"preAuthorizationReview_unknownStatus_subtitle": "Your pre-authorization has been sent to %@",
+	"preAuthorizationReview_unknownStatus_identifier": "Pre-Authorization ID",
+	"preAuthorizationReview_unknownStatus_expiration": "%@ has **%@** to use this pre-authorization",
+	"preAuthorizationReview_unknownStatus_lastCheck": "Checking one last time..",
```

Comment:
This one depends on answer from Slack [thread](https://rdxworks.slack.com/archives/C031A0V1A1W/p1732538607488809)